_data/students.yml

```diff
@@ -60,4 +60,12 @@
   facebook: https://www.facebook.com/aditya.goel.311
   bio: I come here to build.
 
+  name: Gagan Ganapathy
+  github: https://github.com/CodHeK
+  image: gagan.jpg
+  facebook: https://www.facebook.com/GaganGanapathyAs
+  referral: Aditya Goel (Ellusionists)
+  bio: First Contribution here...hoping for more !
```

Comment:
Please add your bio rather :+1: Like who you are , we are planning to make it visible after sometime